### ISSUE TITLE
Webrtc only works on wifi

### ISSUE THREAD

**Issue by filipthornton** [jamalag/webrtc-Part-4#10]
I have built on android an app with video and audio calling. Everything works great if I have the 2 devices connected on the same network (wifi). However if I put one device on mobile data it doesn't work anymore. The connection is made but in videocall the rtcview screen is black and no sound, in audio call I receive no sound on either of the devices. I have to mention that I am using a Turn server. 

Did you encounter this issue ? Can it be because of the TURN server ?

**Comment by jamalag** [comment]
Yes, the issue with mobile data so far seem to require a turn server. Strange that you are using one and the experience is the same.

**Comment by jamalag** [comment]
BTW, when you mention same network, are you using ngrok?
